exec/java-exec/src/main/codegen/templates/DateIntervalFunctionTemplates/ToDateTypeFunctions.java

```diff
@@ -19,9 +19,13 @@
 
 <@pp.dropOutputFile />
 
-<#list dateIntervalFunc.dates as type>
+<#list dateIntervalFunc.varCharToDate as convert>
 
-<@pp.changeOutputFile name="/org/apache/drill/exec/expr/fn/impl/GTo${type}.java" />
+<#if !convert.to?contains("Nullable")>
```

Comment:
-> `<#if convert.to?contains("Nullable")>` and replace the statements inside `if` `else` units

Reply:
Why do not add convert.from to `GTo${convert.to}.java` changeOutputFile name too?